src/transaction/log_record.hpp

```diff
@@ -53,8 +53,8 @@ enum log_rectype
   LOG_CLIENT_USER_POSTPONE_DATA = 11,	/* Obsolete */
   LOG_RUN_NEXT_CLIENT_UNDO = 12,	/* Obsolete */
   LOG_RUN_NEXT_CLIENT_POSTPONE = 13,	/* Obsolete */
-#endif
   LOG_WILL_COMMIT = 14,		/* Transaction will be committed */
```

Comment:
better to edit comment like other obsoleted ones.
```suggestion
  LOG_WILL_COMMIT = 14,		/* Obsolete */
```